lib/react.dart

```diff
@@ -65,6 +65,11 @@ typedef ReactDartFunctionComponentFactoryProxy FunctionComponentRegistrar(DartFu
 /// See: <https://reactjs.org/docs/fragments.html>
 var Fragment = ReactJsComponentFactoryProxy(React.Fragment);
 
+/// Suspense lets you display a fallback UI until its children have finished loading.
+///
+/// See: <https://react.dev/reference/react/Suspense>
+var Suspense = ReactJsComponentFactoryProxy(React.Suspense);
```

Comment:
I know it's not going public yet, but when it does, I still think there should be an extended doc comment with consumption guidelines talking about using `Lazy`, or not if it's not public, so folks can easily find how to convert the JS paradigm to Dart! Let me know (whenever you circle back here) if there was a reason that was included when you addressed the first round feedback!

Reply:
There is a long and lovely doc comment in the over_react version already: https://github.com/Workiva/over_react/pull/820/files#diff-0b321057dde5988d314e11179540acd2207feda3eba1af26f6ac81bd04f603a6R24-R55

eat your heart out

Reply:
> eat your heart out

I would if I felt you addressed my feedback 😜 I won't push it more after this thought, but IMO if we aren't going to put the same level of effort into react-dart as we do OverReact, in terms of comments and API guidelines, then they shouldn't be separate packages. I think the API as is in react-dart is pretty confusing, which is lame for an OSS package

Reply:
bro, same. lol I think react-dart is a waste of time, especially as dart evolves (@greglittlefield-wf probably feels very different) but PERSONALLY it seems like a somewhat unnecessary step between us and the JS, but also there is so much like history there that removing the react-dart dependency would probably break so much shit it would never work lol.

Reply:
The separation of react-dart and over_react is mostly: 
a) tech debt (these started as separate packages, mainly because we didn't own react-dart)
b) organizationally/architecturally, react-dart works well (for the most part) as a separate layer from over_react, handling most of the JS interop, and over_react mostly being the typed props and factory layer on top of that. Even if we were to combine them into a single repo and make most of the react-dart APIs private, we'd probably maintain most of this separation.

The effort required to maintain the two of them separately hasn't been enough at this point to justify the effort to consolidating them. If we find having them be separate is too painful, I'm happy to discuss!

>  but IMO if we aren't going to put the same level of effort into react-dart as we do OverReact, in terms of comments and API guidelines, then they shouldn't be separate packages. I think the API as is in react-dart is pretty confusing, which is lame for an OSS package

That's a fair point. I think the reality here is that we don't expect people to use react-dart directly most of the time, and instead to use over_react. Perhaps to help address this while these packages are separate, we could just be more forthright about how much effort we put into this project's developer experience, and recommend using over_react instead of using this project directly.